src/ansys/additive/core/additive.py

```diff
@@ -77,6 +72,11 @@ class Additive:
         the Additive portion of the Ansys Structures package must be installed.
         This parameter is ignored if the ``server_connections``, ``channel``, or ``host``
         parameter is other than ``None``.
+    product_version: str
+        Version of the Ansys product installation, of the form ``"YYR"``, where ``YY``
```

Comment:
```suggestion
        Version of the Ansys product installation in the form ``"YYR"``, where ``YY``
```
Nice explanation. Probably the best I've seen in my 7 years here!